gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -31,28 +31,26 @@ For more information, see xref:gateway::index.adoc[].
 
 Mule runtime engine includes an embedded Mule Gateway. Mule Gateway enables you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
 
-Using Mule Gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
+Using Mule Gateway, you can, for example:
+
+* Apply a basic authentication policy on top of a Mule application.
+* Enrich an incoming or outgoing message.
+* Add any other complex capability to an API without having to write any code.
 
 If you want to apply policies and generate analytics info, you must use either:
 
-* An API defined in a Mule application.
-* A Mule application that proxies your existing backend implementation.
+* An API defined in a Mule application
+* A Mule application that proxies your existing backend implementation
 
 For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[].
 
 == Flex Gateway vs. Mule Gateway
 
-While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule applications.
-
-Mule runtime engine, based on Java Spring, includes an embedded Mule Gateway. By using this gateway, you can, for example:
-
-* Apply a basic authentication policy on top of a Mule application.
-* Enrich an incoming or outgoing message.
-* Add any other complex capability to an API without having to write any code.
+Flex Gateway can manage and secure APIs running anywhere.
 
-Mule Gateway protects a single Mule API. The key advantage is that it's easy for Mule app developers to provide basic endpoint protection.
+In contrast, Mule Gateway protects a single Mule API. The key advantage is that it's easy for Mule app developers to provide basic endpoint protection.
 
-You can also configure Mule in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot be easily scaled to support high-availability and high-performance applications. Further, Mule consumes more computes or vCores than the smaller-footprint Flex Gateway.
+You can configure Mule in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot easily scale to support high-availability and high-performance applications, which Flex Gateway supports. Further, Mule consumes more computes or vCores than the smaller-footprint Flex Gateway.
```

Comment:
```suggestion
You can configure Mule in Anypoint Runtime Manager as a CloudHub proxy application, protecting multiple backends. However, in this case, the gateway cannot easily scale to support high-availability and high-performance applications, which Flex Gateway supports. Further, Mule consumes more computes or vCores than the smaller-footprint Flex Gateway.
```

Reply:
Is "computes" correct?

Reply:
I think so. "Computes" are units of capacity for vCore processing.